src/Microsoft.ApplicationInsights.AspNet/TelemetryInitializers/UserAgentTelemetryInitializer.cs

```diff
@@ -14,7 +16,8 @@
     /// </summary>
     public class UserAgentTelemetryInitializer : TelemetryInitializerBase
     {
-        public UserAgentTelemetryInitializer(IHttpContextAccessor httpContextAccessor) : base(httpContextAccessor)
+        public UserAgentTelemetryInitializer(IHttpContextAccessor httpContextAccessor, AspNet5EventSource eventSource)
```

Comment:
Have we decided _not_ to use ASPNET's `ILogger` in favor of `EventSource`? Are we _sure_ we want a dependency on the concrete `AspNet5EventSource` class as opposed to an interface that the class implements?